commons/connector/upload/altoida/altoida_contrast_vision_test.avsc

```diff
@@ -0,0 +1,36 @@
+{
+    "namespace": "org.radarcns.connector.upload.altoida",
+    "name": "AltoidaContrastVisionTest",
+    "type": "record",
+    "doc": "Schema of events for the Altoida Tapping test.",
+    "fields": [
+      { "name": "time", "type": "double", "doc": "Timestamp in UTC (s) when the assessment is started by the subject." },
+      { "name": "timeReceived", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+      { "name": "name", "type": "string", "doc": "Questionnaire names." },
+      { "name": "contrastCircles", "type": {
+        "type": "array",
+        "items": {
+          "name": "ContrastCircle",
+          "type": "record",
+          "doc": "Contrast circle screen element.",
+           "fields": [
+             { "name": "id", "type": "int", "doc": "Circle id." },
+             { "name": "radius", "type": "double", "doc": "Circle radius" },
+             { "name": "xCenterOffset", "type": "double", "doc": "Circle location x center offset in cm." },
+             { "name": "yCenterOffset", "type": "double", "doc": "Circle location y center offset in cm." },
+             { "name": "colorAlphaValue", "type": ["null", "string"], "doc": "Circle alpha value in RGBA color space.", "default" : null },
+             { "name": "colorRValue", "type": ["null", "string"], "doc": "Circle red value in RGBA color space.", "default" : null },
+             { "name": "colorGValue", "type": ["null", "string"], "doc": "Circle green value in RGBA color space.", "default" : null },
+             { "name": "colorBValue", "type": ["null", "string"], "doc": "Circle blue value in RGBA color space.", "default" : null }
```

Comment:
Are these hexadecimals (00-ff) or can they be float (0.0-1.0) or decimal (0-255) instead?

Reply:
Yes I checked and they are float. I'll make the changes.